Library/Formula/cv.rb

```diff
@@ -0,0 +1,15 @@
+class Cv < Formula
+  desc "Coreutils Viewer: Show progress for cp, rm, dd..."
```

Comment:
Let's not use `...` in this context - It looks like there should be something else printed afterwards in the command-line environment. Perhaps just `Progress meter for coreutils commands` or something?